### DISCUSSION TITLE
Can't use Codespaces!!!! How to start with codespaces as student?

### DISCUSSION THREAD

**Discussion by Darshan2104** [community/community#28092] [Codespaces]
I recently came to know about codespaces. I want to utilize it but my GitHub account is not showing me an option for codespaces 
What should I do now?
How do I enable codespaces?

Anybody, please help me out

Thanks

**Comment by craiglpeters** [comment]
Thank you for your interest in Codespaces! Later this year we will make Codespaces available to all accounts. But in the mean time it is available for accounts with Teams or Enterprise plan subscriptions. Your instructors can get Teams for free under https://education.github.com/benefits which will give you access to Codespaces.

**Reply by Darshan2104** [reply]
@craiglpeters 
Later this year!!!!!!!
could it be possible to have access to the Beta version?
I've already made a request for it!

Please do needful
Thanks

**Reply by Arafat-jt** [reply]
Hey there @Darshan2104 can you tell me how or atleast to which mail id you wrote the request??

thanks

**Reply by Darshan2104** [reply]
@Arafat-jt just create a ticket at [GitHub Support](https://support.github.com/) and request them for BETA. That's how I got!

**Reply by Arafat-jt** [reply]
I can't, it's either taking me to community forum or documentation
![Screenshot 2022-08-27 155756](https://user-images.githubusercontent.com/63737516/187026219-385ebd75-6b67-48c4-8a11-02cae013fb3d.jpg)

**Reply by Darshan2104** [reply]
@Arafat-jt try this out: https://support.github.com/contact/education
This might works

**Reply by Arafat-jt** [reply]
thanks bro, i was able to create a ticket
let's see how it works out

**Reply by Darshan2104** [reply]
Good luck!

**Comment by jfullstackdev** [comment]
I joined the waitlist last time and I got approved, so I have a free Codespaces by now

**Reply by Darshan2104** [reply]
@jdevstatic I've made a request for a beta version. How long it will take to get approved?

**Reply by jfullstackdev** [reply]
I cannot remember exactly but it's months long, say, 6 months I think so

**Reply by Darshan2104** [reply]
Ohh...man!!!!

**Reply by Darshan2104** [reply]
Thanks

**Reply by jfullstackdev** [reply]
also it's a personal account, no student package involve, 

and it's worth the wait, I can even run Java right away without the complex installation of the local setup

**Reply by Darshan2104** [reply]
that's the thing bro,
I want to do opensource contribution and for that I need to do local setup  which seems quite impossible
one of friend suggested me codespaces and here I'm!

**Reply by jfullstackdev** [reply]
yup, imagine that convenience, see this, I'm coding right now and I did not install Java and anything, I already have the Git and Bash too, it's a remote Linux machine actually

![image](https://user-images.githubusercontent.com/47092464/182825509-2edb2a5e-5b19-4a60-a6b6-a3b31afa1943.png)

if you are not yet familiar with Replit, you can also have that kind of env

https://replit.com/

but Codespaces is for pure dev world while Replit is mostly for students as it is simpler

**Reply by Darshan2104** [reply]
![ice_screenshot_20220804-171425](https://user-images.githubusercontent.com/61326242/182839264-84ab61c0-cfd1-4634-9e85-da527e927f7b.png)

:)

**Reply by jfullstackdev** [reply]
yup, as I mentioned that's mostly for students

![image](https://user-images.githubusercontent.com/47092464/182840224-450510be-dd01-4cbb-927a-66de14f9d705.png)

and they have this pricing

https://replit.com/pricing

**Reply by Darshan2104** [reply]
ohk
Thanks for your support :)

**Comment by ChocolateMagnate** [comment]
Hello @Darshan2104! I am also a student, and I got access to Codespaces by joining the waitlist and I was approved within days. I suspect your student access may be important, because I registered on GitHub with my university account and verified my status with a photo proof, so you may try this way.

**Reply by Darshan2104** [reply]
Actually, I'm no more a student :(
